LR test failures

The LR tests are not run on PRs, only on merge to main. 

They are failing:

https://github.com/AntelopeIO/leap/actions/runs/3108470368/jobs/5037765312
https://github.com/AntelopeIO/leap/actions/runs/3109955801/jobs/5040700386

Broken by: https://github.com/AntelopeIO/leap/pull/199

```
  File "/__w/leap/leap/build/tests/nodeos_retry_transaction_test.py", line 184, in <module>
    assert "overdrawn balance" in str(overdrawtrans), "ERROR: Overdraw transaction attempt should have failed with overdrawn balance."
AssertionError: ERROR: Overdraw transaction attempt should have failed with overdrawn balance.
```

--- a/tests/nodeos_retry_transaction_test.py
+++ b/tests/nodeos_retry_transaction_test.py
@@ -181,7 +181,7 @@ try:
     overdrawTransferAmount = "1001.0000 {0}".format(CORE_SYMBOL)
     Print("Transfer funds %s from account %s to %s" % (overdrawTransferAmount, overdrawAccount.name, cluster.eosioAccount.name))
     overdrawtrans = node.transferFunds(overdrawAccount, cluster.eosioAccount, overdrawTransferAmount, "test overdraw transfer", exitOnError=False, reportStatus=False, retry=1)
-    assert "overdrawn balance" in str(overdrawtrans), "ERROR: Overdraw transaction attempt should have failed with overdrawn balance."
+    assert overdrawtrans is None, f"ERROR: Overdraw transaction attempt should have failed with overdrawn balance: {overdrawtrans}"
 
     def cacheTransIdInBlock(transId, transToBlock, node):
         global lastIrreversibleBlockNum

--- a/tests/testUtils.py
+++ b/tests/testUtils.py
@@ -204,7 +204,6 @@ class Utils:
         (output,error)=popen.communicate()
         Utils.checkOutputFileWrite(start, cmd, output, error)
         if popen.returncode != 0 and not ignoreError:
-            Utils.Print("ERROR: %s" % error)
             # for now just include both stderr and stdout in output, python 3.5+ supports both a stdout and stderr attributes
             raise subprocess.CalledProcessError(returncode=popen.returncode, cmd=cmd, output=error+bytes(" ", 'utf-8')+output)
         return output.decode("utf-8")

